src/api/pdc_client_connect.c

```diff
@@ -7454,6 +7409,67 @@ PDC_Client_query_kvtag_col(const pdc_kvtag_t *kvtag, int *n_res, uint64_t **pdc_
     FUNC_LEAVE(ret_value);
 }
 
+perr_t
+PDCtag_delete(pdcid_t obj_id, char *tag_name, int is_cont)
```

Comment:
Can you make sure this function has documentation on top of it?

Reply:
Changed it to a static function, public functions have been properly documented in the header files.